test/jobs/parse_report_job_test.rb

```diff
@@ -8,13 +8,17 @@ class ParseReportJobTest < ActiveSupport::TestCase
     @parse_report_job = ParseReportJob.new
     @file = file_fixture('report.tar.gz').read
     @parser = mock('XccdfReportParser')
+    @issue_id = 'ssg:rhel7|short_profile_ref_id|rule_ref_id'
   end
 
   test 'payload tracker is notified about successful processing' do
```

Comment:
It would be better to duplicate this test only for `RemediationUpdates` and name it accordingly. That way if one of PayloadTracker or RemediationsUpdates is broken in the future, it will be clear by the failing test which one isn't working as expected.

Reply:
Good suggestion.